### PR TITLE
mdraid: add option description for OCF_CHECK_LEVEL

### PR DESCRIPTION
**Pull request by zhaohem**
Signed-off-by: Heming Zhao <heming.zhao@suse.com>

### CODE REVIEW

File: heartbeat/mdraid [hunk 1]
```diff
@@ -67,6 +67,14 @@ It uses mdadm(8) to start, stop, and monitor the MD devices.
 Supported clustered (i.e., clonable active-active) arrays are linear,
 raid0, and clustered raid1/raid10 (i.e. mdadm(8) created with
 --bitmap=clustered).
+
+Option: OCF_CHECK_LEVEL
+
+To set OCF_CHECK_LEVEL any number greater than 0, the standard monitor
```
Comment (oalbrigt) [inline heartbeat/mdraid:73]: Can you update the first part of the sentence to "When OCF_CHECK_LEVEL is set to any number greater than 0,"?

Just minor nitpicking that makes it easier to understand without re-reading.

**Comment by zhaohem** [comment]
force pushed a new version

**Comment by oalbrigt** [comment]
ok to test

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**